Filtration.Parser/Services/ItemFilterBlockTranslator.cs

```diff
@@ -536,13 +591,13 @@ public string TranslateItemFilterBlockToString(IItemFilterBlock block)
                 {
                     outputString += (!block.Enabled ? _disabledNewLine : _newLine) + blockItem.OutputText;
                 }
-            }
-
-            if (!block.Enabled)
-            {
-                outputString += Environment.NewLine +  "#Disabled Block End";
-            }
-            
+            }
+
+            // Replace 'Maelström' to prevent encoding problems in other editors
+            outputString.Replace("Maelström Staff", "Maelstr");

```

Comment:
Doesn't this change filters in unintentional ways? For example if someone wanted to hide "Maelström Staff" but then it's changed to "Maelstr" when saving the filter, it'd end up hiding all 3 items that start with Maelström?

Reply:
Correct. This was added because Neversink told me that the word 'Maelström' may cause encoding errors in site and I wanted to make Filterblade scripts edited with Filtration to be compatible with the site. However the problem shouldn't be neglected so  I will think of something else or remove it until a better solution comes to mind.